guides/getting-started.md

```diff
@@ -1,5 +1,5 @@
 # Getting Started
-###### Last updated February 20, 2018
+###### Last updated March 10, 2018
```

Comment:
Do we really need a "last updated"?  The Git commit history for the file should be sufficient, right?
I have seen many times in the past that people tend to forget to change the "last updated" later on down the road, making it less useful.

Reply:
@andrew-frueh introduced it, so I would create an issue and discuss it with him

Reply:
@andrew-frueh just for future reference, we could always add this to the MD file during the build before it is published using a command like this:

```bash
git log -1 --format="%ad" --date=short -- guides/getting-started.md
```

```
2018-03-03
```

This way we could not have to worry about updating the date or making sure the date isn't out of date.  We should probably make a separate issue of this if you are open to the idea.